.github/workflows/release.yml

```diff
@@ -119,7 +119,7 @@ jobs:
           rm get-pip.py
 
       - name: Checkout Kani
-        uses: actions/checkout@v3
+        run: git clone https://github.com/model-checking/kani/
```

Comment:
This will clone the repository under `${workspace}/kani/`, while other actions expect the clone to be in `${workspace}`.